Explorer/Assets/Scripts/ECS/Unity/Visibility/Systems/VisibilitySystemBase.cs

```diff
@@ -33,28 +33,28 @@ protected override void Update(float t)
         private void ProcessEvent(Entity entity, TComponent @event)
         {
             if (World.TryGet(entity, out PBVisibilityComponent? visibilityComponent))
-                UpdateVisibilityInternal(@event, visibilityComponent!.GetVisible());
+                UpdateVisibilityInternal(ref @event, visibilityComponent!.GetVisible());
```

Comment:
it's wrong by design to use `event` by `ref` as it's written always as a copy. That's why it was `in` before. You can't change it.